parkings/migrations/0029_enforcement_domain.py

```diff
@@ -14,16 +15,57 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
+        migrations.CreateModel(
+            name='EnforcementDomain',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='time created')),
+                ('modified_at', models.DateTimeField(auto_now=True, verbose_name='time modified')),
+                ('code', models.CharField(max_length=10, unique=True, verbose_name='code')),
+                ('name', models.CharField(max_length=40, verbose_name='name')),
+                ('geom', django.contrib.gis.db.models.fields.MultiPolygonField(
+                    srid=3879, verbose_name='geometry', default="MULTIPOLYGON EMPTY"
+                )),
+            ],
+            options={
+                'ordering': ('code',),
+            },
+        ),
+        migrations.CreateModel(
+            name='Enforcer',
+            fields=[
+                ('created_at', models.DateTimeField(auto_now_add=True, verbose_name='time created')),
+                ('modified_at', models.DateTimeField(auto_now=True, verbose_name='time modified')),
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('name', models.CharField(max_length=80, verbose_name='name')),
+                ('enforced_domain', models.ForeignKey(help_text='An enforcement domain than is enforced by this enforcer', null=True, on_delete=django.db.models.deletion.PROTECT, to='parkings.EnforcementDomain', verbose_name='enforced domain')),
```

Comment:
There seems to be a typo in this help_text (probably written by me :smiley_cat:). The `than` word should be `that`, of course.

Reply:
Remove the `null=True, blank=True` part, since every enforcer should have an ED.

And since there always is a single enforcement domain for each Enforcer, the `An` article in the `help_text` should be replaced with `The`, i.e. the whole help text should be: "The enforcement domain enforced by this enforcer".